assets/js/modal.js

```diff
@@ -34,6 +34,14 @@ function renderModal () {
   qs(MODAL_CLOSE_BUTTON_SELECTOR).addEventListener('click', event => {
     closeModal()
   })
+
+  qs(MODAL_SELECTOR).addEventListener('click', event => {
+    const classList = event.target.classList
+    // if we clicked on the modal overlay/parent but not the modal content
+    if (classList.contains('modal') && classList.contains('shown') && classList.length === 2) {
```

Comment:
Should we check if the ID is "modal" instead of hardcoding classes?

Reply:
Sorry for the delayed response! Yeah that would be a better way to go about things. I can refactor the code and make a new PR for that using the id instead

Reply:
Ah nevermind I see that it's been done already. Awesome!